modules/diffusion/boundary.h

```diff
@@ -29,6 +30,15 @@ class Boundary
   std::array<double, 3> values_ = {0.0, 0.0, 0.0};
 };
 
+/** Multigroup diffusion boundary */
+class MGBoundary
+{
+public:
+  BoundaryType type_ = BoundaryType::Vacuum;
+
+  std::array<std::vector<double>, 3> mg_values_;
+};
```

Comment:
Ultimately, it would be nice to have a group agnostic diffusion solver. We should consider redoing the diffusion solvers in the future.

Reply:
I think this data structure is only used to store the user input from the lua file. Then, it is internally converted into what ever the solver needs. Am I right @ragusa or did I miss something?

Reply:
this is the (a,b,c) coefficients of the Robin BC in multigroup

Reply:
Right, this is the general form of a Robin BC for multigroup. It could be used for one-group instead of having a separate boundary condition. This is not something to change now. Just jotting down thoughts so I am less likely to lose them in the future.